crm_lead_to_event_registration/models/crm_lead.py

```diff
@@ -1,18 +1,17 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from openerp import _, api, exceptions, fields, models
 
 
 class CrmLead(models.Model):
-    _inherit = "crm.lead"
+    _inherit = 'crm.lead'
```

Comment:
Why changing quotes if they are not needed. Although there can be a style preference, if the code is already written, it's better to not touch that lines for reducing the diff size